src/flextopo_model.jl

```diff
@@ -0,0 +1,791 @@
+        sel = inds,
+        defaults = 0.2,
+        type = Float,
+        dimname = :classes,
+    )
+    fmax =
+        ncread(
+            nc,
+            config,
+            "vertical.fmax";
+            sel = inds,
+            defaults = 2.0,
+            type = Float,
+            dimname = :classes,
+        ) .* (Δt / basetimestep)
+    shmin = ncread(
+        nc,
+        config,
+        "vertical.shmin";
+        sel = inds,
+        defaults = 0.2,
+        type = Float,
+        dimname = :classes,
+    )
+    kmf = ncread(
+        nc,
+        config,
+        "vertical.kmf";
+        sel = inds,
+        defaults = 1.0,
+        type = Float,
+        dimname = :classes,
+    )
+    srmax = ncread(
+        nc,
+        config,
+        "vertical.srmax";
+        sel = inds,
+        defaults = 260.0,
+        type = Float,
+        dimname = :classes,
+    )
+    beta = ncread(
+        nc,
+        config,
+        "vertical.beta";
+        sel = inds,
+        defaults = 0.3,
+        type = Float,
+        dimname = :classes,
+    )
+    lp = ncread(
+        nc,
+        config,
+        "vertical.lp";
+        sel = inds,
+        defaults = 0.3,
+        type = Float,
+        dimname = :classes,
+    )
+    perc =
+        ncread(
+            nc,
+            config,
+            "vertical.perc";
+            sel = inds,
+            defaults = 0.30,
+            type = Float,
+            dimname = :classes,
+        ) .* (Δt / basetimestep)
+    cap =
+        ncread(
+            nc,
+            config,
+            "vertical.cap";
+            sel = inds,
+            defaults = 0.20,
+            type = Float,
+            dimname = :classes,
+        ) .* (Δt / basetimestep)
+    kf =
+        ncread(
+            nc,
+            config,
+            "vertical.kf";
+            sel = inds,
+            defaults = 0.1,
+            type = Float,
+            dimname = :classes,
+        ) .* (Δt / basetimestep)
+    alfa = ncread(
+        nc,
+        config,
+        "vertical.alfa";
+        sel = inds,
+        defaults = 1.0,
+        type = Float,
+        dimname = :classes,
+    )
+    ds = ncread(
+        nc,
+        config,
+        "vertical.ds";
+        sel = inds,
+        defaults = 0.2,
+        type = Float,
+        dimname = :classes,
+    )
+
+    # read x, y coordinates and calculate cell length [m]
+    y_nc = read_y_axis(nc)
+    x_nc = read_x_axis(nc)
+    y = permutedims(repeat(y_nc, outer = (1, length(x_nc))))[inds]
+    cellength = abs(mean(diff(x_nc)))
+
+
+    sizeinmetres = get(config.model, "sizeinmetres", false)::Bool
+    xl, yl = cell_lengths(y, cellength, sizeinmetres)
+
+
+    # snow = zeros(Float, nclass, n)
+    # snowwater = zeros(Float, nclass, n)
+    interceptionstorage = zeros(Float, nclass, n)
+    hortonpondingstorage = zeros(Float, nclass, n)
+    hortonrunoffstorage = zeros(Float, nclass, n)
+    rootzonestorage = zeros(Float, nclass, n)
```

Comment:
In VSCode `rootzonestorage` here is greyed out, because it is not used, same as the next two lines. The values in the end seem to be copied from `srmax`. Would be good to remove these lines, as well as the other commented out code lines below and in flextopo.jl that you ended up not using.

Reply:
yes indeed, I have cleaned up this part and other commented out lines in commit 7dde6ff6469d2ec5e4b975f9dd0f455987dff313